dotnet-roslyn/config/Audacia.CodeAnalysis/Base/.editorconfig

```diff
@@ -2233,4 +2233,283 @@ dotnet_diagnostic.IDE0051.severity = none
+dotnet_diagnostic.IDE0062.severity = warning
+
+# Use simple using statement
+dotnet_diagnostic.IDE0063.severity = warning
+
+# Make struct fields writable
+dotnet_diagnostic.IDE0064.severity = warning
+
+# using directive placement
+dotnet_diagnostic.IDE0065.severity = warning
+
+# Use switch expression
+dotnet_diagnostic.IDE0066.severity = warning
+
+# Use System.HashCode.Combine
+dotnet_diagnostic.IDE0070.severity = warning
+
+# Simplify interpolation
+dotnet_diagnostic.IDE0071.severity = warning
+
+# Add missing cases to switch expression
+dotnet_diagnostic.IDE0072.severity = warning
+
+# Use file header
+dotnet_diagnostic.IDE0073.severity = warning
+
+# Use coalesce compound assignment
+dotnet_diagnostic.IDE0074.severity = warning
+
+# Simplify conditional expression
+dotnet_diagnostic.IDE0075.severity = warning
+
+# Remove invalid global SuppressMessageAttribute
+dotnet_diagnostic.IDE0076.severity = warning
+
+# Avoid legacy format target in global SuppressMessageAttribute
+dotnet_diagnostic.IDE0077.severity = warning
+
+# Use pattern matching
+dotnet_diagnostic.IDE0078.severity = warning
+
+# Remove unnecessary suppression
+dotnet_diagnostic.IDE0079.severity = warning
+
+# Remove unnecessary suppression operator
+dotnet_diagnostic.IDE0080.severity = warning
+
+# Remove ByVal
+dotnet_diagnostic.IDE0081.severity = warning
+
+# Convert typeof to nameof
+dotnet_diagnostic.IDE0082.severity = warning
+
+# Use pattern matching (not operator)
+dotnet_diagnostic.IDE0083.severity = warning
+
+# Use pattern matching (IsNot operator)
+dotnet_diagnostic.IDE0084.severity = warning
+
+# Simplify new expression
+dotnet_diagnostic.IDE0090.severity = warning
+
+# Remove unnecessary equality operator
+dotnet_diagnostic.IDE0100.severity = warning
+
+# Remove unnecessary discard
+dotnet_diagnostic.IDE0110.severity = warning
+
+# Simplify LINQ expression
+dotnet_diagnostic.IDE0120.severity = warning
+
+# Namespace does not match folder structure
+dotnet_diagnostic.IDE0130.severity = warning
+
+# Simplify object creation
+dotnet_diagnostic.IDE0140.severity = warning
+
+# Prefer null check over type check
+dotnet_diagnostic.IDE0150.severity = warning
+
+# Use block-scoped namespace
+dotnet_diagnostic.IDE0160.severity = warning
+
+# Use file-scoped namespace
+dotnet_diagnostic.IDE0161.severity = warning
+
+# Simplify property pattern
+dotnet_diagnostic.IDE0170.severity = warning
+
+# Use tuple to swap values
+dotnet_diagnostic.IDE0180.severity = warning
+
+# Remove unnecessary lambda expression
+dotnet_diagnostic.IDE0200.severity = warning
+
+# Convert to top-level statements
+dotnet_diagnostic.IDE0210.severity = warning
+
+# Convert to 'Program.Main' style program
+dotnet_diagnostic.IDE0211.severity = warning
+
+# Add explicit cast in foreach loop
+dotnet_diagnostic.IDE0220.severity = warning
+
+# Use UTF-8 string literal
+dotnet_diagnostic.IDE0230.severity = warning
+
+# Use conditional delegate call
+dotnet_diagnostic.IDE1005.severity = warning
```

Comment:
nit: Can `IDE0005` and `IDE0006` be place in the correct positional numerically in these configs

Reply:
Not sure if I'm missing something, but there isn't an rule labelled `IDE0006`

Reply:
Ah yep sorry, I misread that these two rules start with a `1` so are in the correct order